utils.c

```diff
@@ -250,6 +256,7 @@ static const struct luaL_Reg torch_utils__ [] = {
   {"pointer", luaT_lua_pointer},
   {"setheaptracking", torch_setheaptracking},
   {"updateerrorhandlers", torch_updateerrorhandlers},
+  {"getmemory", torch_getmemory},
```

Comment:
I agree that simply `getmemory` is not a good name.
`memoryused` is indeed better.
Any other opinions ? `cutorch` has a `getMemoryUsage` (which prints the total memory available and the memory used for the GPU). Maybe it would be good to have some uniformity in there.